### PR TITLE
page centered & added outlined border

### PR DESCRIPTION
**Pull request by egatsak**
border outlined (left and right)
page align became center

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,76 +1,84 @@
...
 
 * {
-    box-sizing: border-box;
-  }
+  box-sizing: border-box;
+}
 
 body {
-    font-family: 'Franklin Gothic Medium', 'Arial Narrow', serif;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", serif;
```
Comment (Vadimchesh) [inline style.css:18]: Add background-color: whitesmoke and delete it from .bordero

File: style.css [hunk 2]
```diff
@@ -1,76 +1,84 @@
...
 }
 
 .bordero {
-    background-color: whitesmoke;
-    width: 1000px;
-    border: 2px solid rgba(250, 50, 50, .4);
-    padding: 100px auto 50px;
+  margin: auto;
```
Comment (Vadimchesh) [inline style.css:22]: add vertical margin to the border

**Comment by Vadimchesh** [comment]
@egatsak could you make `Contacts` and `About` as a link with `href` is `#`

**Comment by Vadimchesh** [comment]
@egatsak also you can replace `href` value in a logo with `#` instead of `index.html

File: style.css (hunk 2 above)
Comment (egatsak) [reply style.css:22]: done

File: style.css (hunk 1 above)
Comment (egatsak) [reply style.css:18]: done

**Comment by Vadimchesh** [comment]
LGFM (Look good for me) 
You can merge

**Pull request merged**